test/usage/dune-sigterm/test-daemon.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+dune exec ./daemon.exe &
+# Wait a bit until the application has started
+sleep 1
```

Comment:
it is a bit unfortunate that i have to do this, but if i do not sleep it seems the application does not have enough time to start up and install the signal handler. putting a lower value for sleep does not seem to work very well. suggestions? i could have `daemon.exe` communicate with the outside world when it has started (write a file or something) and wait for that instead.

Reply:
I think it's ok. Alternatively, would the corrupted file happens if you have an OCaml code that does a `fork` and kill its child?

Reply:
`sleep 1` is fine for this test.